modules/apps/foundation/frontend-theme/responsive-boat-theme/src/js/main.js

```diff
@@ -0,0 +1,44 @@
+
+// This function gets loaded when everything, including the portlets, is on the page
+
+Liferay.on('allPortletsReady', myPageJS);
+
+// Function Containing All of My Event Handlers
+
+function myPageJS() {
+	let nav = $('#navigation');
```

Comment:
If you are using jQuery, you should use the jQuery that is included with Liferay using ```AUI.$();```.